src/app/map/components/MapContainer.js

```diff
@@ -79,29 +97,124 @@ class MapContainer extends Component {
     this.state = {
       // Store containers list from backend. Each container has "id", "lat" and "lng"
       containers: [],
+      selectedId: 0,
+      selectedLon: 0,
+      selectedLat: 0,
+      infoContainer: '',
+      load: true,
     };
     this.getData = getData.bind(this);
+    this.showInfo = this.showInfo.bind(this);
   }
 
   componentDidMount() {
     this.getData();
+    this.showInfo('0');
```

Comment:
selectedId is a number, not a string. Please change this.showInfo('0') to this.showInfo(0). This must go along with the change suggested in line 261.